aten/src/ATen/native/native_functions.yaml

```diff
@@ -5660,13 +5660,13 @@
     CPU: rrelu_with_noise_cpu
     CUDA: legacy::cuda::_thnn_rrelu_with_noise_forward
 
-- func: rrelu_with_noise_backward.grad_input(Tensor grad_output, Tensor self, Tensor noise, Scalar lower, Scalar upper, bool training, *, Tensor(a!) grad_input) -> Tensor(a!)
+- func: rrelu_with_noise_backward.grad_input(Tensor grad_output, Tensor self, Tensor noise, Scalar lower, Scalar upper, bool training, bool inplace=False, *, Tensor(a!) grad_input) -> Tensor(a!)
```

Comment:
I did a grep through the codebase and I don't see a reason why `rrelu_with_noise_backward.grad_input` or `leaky_relu_backward.grad_input` should exist as operators; they're not used anywhere except for their out-of-place variants. It looks like there were some historical reasons for their existence: back in the day, the THNN CPU/CUDA versions of this function were separate and we used dispatch to call out to them. However, those operators have since been migrated into ATen.

Since we're modifying these lines anyways, we could do the following as a clean-up task:
- delete these two operators from native_functions.yaml
- Call `native::rrelu_with_noise_backward_out` directly from `native::rrelu_with_noise_backward` (and same for `leaky_relu_backward`)

Reply:
removed out version backward functions